packages/react/src/components/Authenticator/SignIn/SignIn.tsx

```diff
@@ -61,6 +61,9 @@ export function SignIn() {
             className="amplify-flex"
             disabled={isPending}
           >
+            <VisuallyHidden>
+              <legend>Sign in</legend>
```

Comment:
Forgot to mention that this should be translated. See example here:
https://github.com/aws-amplify/amplify-ui/blob/57f1441e4809218a813148d0942de8171d159831/packages/react/src/components/Authenticator/ConfirmSignUp/ConfirmSignUp.tsx#L60

Reply:
Thanks for catching it. Updated